ambry-protocol/src/main/java/com.github.ambry.protocol/BlobStoreControlRequestType.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.protocol;
+
+/**
+ * Enum of types of BlobStore control request.
+ * The order of these enums should not be changed since their relative position goes into the serialized form of
+ * requests.
+ */
+public enum BlobStoreControlRequestType {
```

Comment:
`BlobStoreControlAction`?

Reply:
makes more sense, will change as you suggested.